string display_name
string racecar_color
---
bool success